modules/virt-attaching-virtio-disk-to-windows.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms/virt-installing-qemu-guest-agent.adoc
+
+:_content-type: PROCEDURE
+[id="virt-attaching-virtio-disk-to-windows_{context}"]
+= Attaching VirtIO container disk to Windows VMs during installation
+
+You must attach the VirtIO container disk to the Windows VM to install the necessary Windows drivers. This can be done during creation of the VM or to an existing VM.
+
+.Procedure
+
+. When creating a Windows VM xref:../creating_vms_rh/virt-creating-vms-from-templates.html#virt-creating-vm-from-template_virt-creating-vms-from-templates[from a template], click *Customize VirtualMachine*.
+. Select the checkbox to enable the *Mount Windows drivers disk* option.
+. Click the *Customize VirtualMachine parameters* button.
```

Comment:
With regard to minimalism, no need to mention "button" unless there are two controls in the UI with the same name and you need to distinguish.